src/norm/sqlite.nim

```diff
@@ -364,16 +365,44 @@ proc selectOneToMany*[O: Model, M: Model](dbConn; oneEntry: O, relatedEntries: v
   dbConn.select(relatedEntries, sqlCondition, oneEntry.id)
 
 proc selectOneToMany*[O: Model, M: Model](dbConn; oneEntry: O, relatedEntries: var seq[M]) =
-  ## A convenience proc. Fetches all entries of a "many" side from a one-to-many 
-  ## relationship between the model of `oneEntry` and the model of `relatedEntries`.
-  ## The field used to fetch the `relatedEntries` is automatically inferred as long
-  ## as the `relatedEntries` model has only one field pointing to the model of 
-  ## `oneEntry`. Will not compile if `relatedEntries` has multiple fields that 
-  ## point to the model of `oneEntry`. Specify the `foreignKeyFieldName` parameter 
-  ## in such a case.
+  ## A convenience proc. Fetches all entries of a "many" side from the single one-to-many 
+  ## relationship between the model of `oneEntry` and the model of `relatedEntries`
+  ## puts them into `relatedEntries`. The field used to fetch the `relatedEntries` 
+  ## is automatically inferred as long as the `relatedEntries` model has only one 
+  ## field pointing to the model of `oneEntry`. 
+  ## Will not compile if `relatedEntries` has multiple fields that point to the 
+  ## model of `oneEntry`. Specify the `foreignKeyFieldName` parameter in such a
+  ## case.
   const foreignKeyFieldName: string = M.getRelatedFieldNameTo(O)
   selectOneToMany(dbConn, oneEntry, relatedEntries, foreignKeyFieldName)
 
+proc selectOneToMany*[O: Model, M: Model](dbConn; oneEntries: seq[O], relatedEntries: var Table[int64, seq[M]], foreignKeyFieldName: static string) =
+  ## Fetches all entries of multiple "many" side from multiple one-to-many relationships
+  ## between the entries within `oneEntries` and the model of `relatedEntries`. This is 
+  ## done with a single query to the database. The various many-to-one relationships are
+  ## split into a table, where the id of each entry in `oneEntries` is mapped to the entries
+  ## pointing to it. It is ensured at compile time that the field specified here is a 
```

Comment:
This is the first of 2 key procs for this PR!
This allows you to query the one-to-many relationships of multiple entries at once!